src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -205,22 +211,37 @@ export const decryptPlaintext = (encryptedPlaintext: TransformedEncryptedMessage
     });
 
 /**
- * Create a message signature of the current time, segment ID, user ID, and public signing key. Encode that as a base64 string and sign it
- * using ed25519.
+ * Create our complex request signature using the provided paramters. Details about how this signature works are written up in
+ * https://github.com/IronCoreLabs/design-docs/issues/369
```

Comment:
If you'd rather link to or copy in something from a public repo you could reference https://github.com/IronCoreLabs/ironoxide/issues/50

Reply:
Oh good call, I wasn't even thinking about that repo being private, thanks!